src/omero_zarr/raw_pixels.py

```diff
@@ -243,7 +243,11 @@ def plate_to_zarr(plate: omero.gateway._PlateWrapper, args: argparse.Namespace)
     if acquisitions:
         plate_acq = [marshal_acquisition(x) for x in acquisitions]
 
-    for well in plate.listChildren():
+    wells = plate.listChildren()
+    # sort by row then column...
+    wells = sorted(wells, key=lambda x: (x.row * 100 + x.column))
```

Comment:
Where does `100` from from here? Would just returning the tuple `(x.row, x.column)` get you what you need?

Reply:
Ah - didn't know you could return a tuple for sorting. That works fine - fixed in [cdc22fd](https://github.com/ome/omero-cli-zarr/pull/134/commits/cdc22fd88043558bed346abe3ed6183a193f424e)